_sass/components/_accordions.scss

```diff
@@ -0,0 +1,65 @@
+// Accordions
+//
+// Togglable accordion. Button is visible only in 
+// screen sizes smaller than medium
+//
+// Markup: 
+// <ul accordion>
+//    <li accordion-item accordion-open="true">
+//        <h3>Heading</h3>
+//        <button class="accordion-button" accordion-button></button>
+//        <div class="accordion-content">Accordion Content</div
+//    </li>
+// </ul>
+//
+// Styleguide components.accordions
+
+[accordion-item] {
+  position: relative;
+
+  &[accordion-open=true] {
+    .accordion-button:after {
+      content: "\f077";
+    }
+
+    .accordion-content {
+      display: inline-block;
+    }
+  }
+
+  &[accordion-open=false] {
+    .accordion-button:after {
+      content: "\f078";
+    }
+    .accordion-content {
+      display: none;
+      @include respond-to(medium-up) {
+        display: inline-block;
+      }
+    }
+  }
+
+
+}
+
+.accordion-button {
+  background-size: 50%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background-color, background-size, height, margin, position, right, width